### PR TITLE
completed javascript-arrays

### PR DESCRIPTION
**Pull request by kyle-foral**
image

### CODE REVIEW

File: javascript-arrays/main.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+const colors = ['red', 'white', 'blue'];
+console.log('Value of colors[0] ', colors[0]);
+console.log('Value of colors[1] ', colors[1]);
+console.log('Value of colors[2] ', colors[2]);
+
+const America = 'America is ' + colors[0] + ', ' + colors[1] + ', ' + colors[2] + '.';
```
Comment (shawnkost) [inline javascript-arrays/main.js:6]: Variable names should always start with lowercase! JavaScript uses camelCase. Also you're missing an `and`! Compare your output to the example output in the instructions.

File: javascript-arrays/main.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+console.log(America);
+
+colors[2] = 'green';
+const Mexico = 'Mexico is ' + colors[0] + ', ' + colors[1] + ', ' + colors[2] + '.';
```
Comment (shawnkost) [inline javascript-arrays/main.js:10]: Same notes as above regarding variable name and missing `and` ^

File: javascript-arrays/main.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+console.log(Mexico);
+
+console.log('Value of colors ', colors);
+
+const students = ['Chase', 'Kaine', 'Spencer', 'Cameron'];
+const numberOfStudents = students.length;
+
+console.log('There are ', numberOfStudents, ' students in the class.');
```
Comment (shawnkost) [inline javascript-arrays/main.js:18]: You should use a `+` addition operator here when concatenating multiple strings.

File: javascript-arrays/main.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+const lastIndex = numberOfStudents - 1;
+const lastStudent = students[lastIndex];
+console.log('The last student in class is ', lastStudent);
```
Comment (shawnkost) [inline javascript-arrays/main.js:22]: Compare this sentence to the sentence in our example output in the instructions!

**Review by shawnkost — CHANGES_REQUESTED** [review]
Looking good so far Kyle! Just a few changes requested here!

File: javascript-arrays/main.js (hunk 4 above)
Comment (kyle-foral) [reply javascript-arrays/main.js]: image

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**